src/storybook/components/anchor-list-item/anchor-list-item.scss

```diff
@@ -1,5 +1,5 @@
 @import "../../../styles/global-css-settings.scss";
-@import "../styles/typography";
+@import "../../styles/typography.scss";
```

Comment:
moved the story styles outside of the storybook/components folder to the storybook folder